### PR TITLE
Replace travis checks with GH actions

### PR DESCRIPTION
**Pull request by rflperry**
Travis recently changed to reduce free open source software credits. Moving version and test checks to circleci here.

I'm now thinking though that github actions can replace the travis functionality.

### CODE REVIEW

**Comment by agramfort** [comment]
@rflperry I would do this with GH actions. I would actually move completely to the combo GH actions for unit tests  and circleci for the sphinx doc

File: README.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 
 [![PyPI - Python Version](https://img.shields.io/pypi/pyversions/mvlearn.svg)](https://img.shields.io/pypi/pyversions/mvlearn.svg)
-[![Build Status](https://travis-ci.com/mvlearn/mvlearn.svg?branch=main)](https://travis-ci.com/mvlearn/mvlearn)
```
Comment (agramfort) [inline README.md:3]: can you add the GH action badge?

**Comment by rflperry** [comment]
Seems good for now as the necessary minimum

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
Thx @rflperry

**Pull request merged**